src/consume.ts

```diff
@@ -389,6 +389,13 @@ function messageViewerStartPollingCommand(
             }
             default: {
               reportable = { message: "Something went wrong." };
+              window
+                .showErrorMessage("Error response while consuming messages.", "Open Logs")
+                .then((action) => {
+                  if (action === "Open Logs") {
+                    commands.executeCommand("confluent.showSidecarOutputChannel");
+                  }
+                });
```

Comment:
I don't know how much this is user friendly generally, but should be a good thing to have while we troubleshooting stuff.

Reply:
Agreed; not actionable but should at least give users a decent hint what to report